build.gradle

```diff
@@ -1,6 +1,8 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 
 buildscript {
+    ext.mozilla_components_version = '0.10'
```

Comment:
This can be removed when the version is set inline in the dependency declararion